build.gradle

```diff
@@ -83,34 +84,23 @@ subprojects {
         skipConfigurations = ['clover', 'jmh', 'spotbugs']
         format = 'ALL'
         failBuildOnCVSS = 4.0
-    }
-
-    spotbugs {
-        sourceSets = [sourceSets.main]
+        cve {
+            urlModified = "https://freedumbytes.gitlab.io/setup/nist-nvd-mirror/nvdcve-1.1-modified.json.gz"
+            urlBase = "https://freedumbytes.gitlab.io/setup/nist-nvd-mirror/nvdcve-1.1-%d.json.gz"
+        }
```

Comment:
This is to work around the broken NVD files? Worth a comment to note that.

Reply:
Yes, it's broken since yesterday. I will add a comment.